tools/pullJSONtoLDB.mjs

```diff
@@ -1,14 +1,28 @@
 import {compilePack} from "@foundryvtt/foundryvtt-cli";
 import {promises as fs} from "fs";
+import path from "path";
 
 const MODULE_ID = process.cwd();
```

Comment:
We should probably take this opportunity to rename this variable to `SYSTEM_ID` - I had copied the original code from my 5e content packs.